TAGradingServer/account/account-rubric.php

```diff
@@ -37,9 +37,12 @@
     $icon = '<i class="icon-remove icon-white"></i>';
     $color  = "#DA4F49";
     $icon_color  = "#DA4F49";
-    if ($eg->active_assignment == 0) {
+    if ($eg->active_assignment == -1) {
         $part_status  = 'Cancelled';
     }
+    else if($eg->active_assignment == 0){
+        $part_status = 'No Submission';
+    }
```

Comment:
This should be reversed. When you cancel an assignment, its active_version is set to 0 and that's what's going to be parsed as. -1 would only happen if that was the default value for active_assignment was -1.

Also, I'd prefer if you renamed this to "active_version" throughout (also changing [this line](https://github.com/Submitty/Submitty/blob/gradeable_improvements/TAGradingServer/app/models/ElectronicGradeable.php#L325) to read "active_version" as that's what the field is now, not active_assignment in that json)